ThreadJob/ThreadJob/PSThreadJob.cs

```diff
@@ -438,6 +441,12 @@ private ThreadJob()
             this.Debug = _ps.Streams.Debug;
             this.Debug.EnumeratorNeverBlocks = true;
 
+            if (s_HasInformationStream)

```

Comment:
I should have mentioned it before, but this does not solve the problem.  The .Net jitter still throws a runtime error because the Information property does not exist on DL platforms.  I am not aware of any solution other than targeting the module for different versions of PowerShell which I would rather not do.